### ISSUE TITLE
Does attr support nest schema?

### ISSUE THREAD

**Issue by MacHu-GWU** [python-attrs/attrs#255]
Wondering how to do nesting with attr:

```python
@attr.s
class Profile(object):
    ssn = attr.ib()

@attr.s
class Person(object):
    name = attr.ib()
    profile = ??? #<--- What should I do

person_data = {
    "name": "Jack",
    "profile": {"ssn": "123-456-789}
}

# Obviously Not Working, BUT this should be the desired way to do it
# or: person = Person.load(person_data)
Person(**person_data) 

# This does work, but looks dumb
Person(name=person_data["name"], profile=Profile(**person_data["profile"])
```

**Comment by RonnyPfannschmidt** [comment]
You could use the instance_of validator

**Comment by MacHu-GWU** [comment]
@RonnyPfannschmidt But how its been load from data, I updated the issue.

**Comment by hynek** [comment]
attrs doesn't have support for serialization *yet*.  You may want to look at [cattrs](https://github.com/Tinche/cattrs).

**Comment by hynek** [comment]
fixed in #370
